modules/openxr/openxr_api.cpp

```diff
@@ -1787,6 +1788,31 @@ void OpenXRAPI::end_frame() {
 	}
 }
 
+real_t OpenXRAPI::get_display_refresh_rate() const {
+	OpenXRDisplayRefreshRateExtension *drrext = OpenXRDisplayRefreshRateExtension::get_singleton();
+	if (drrext) {
+		return drrext->get_refresh_rate();
+	}
+
+	return 0.0;
+}
+
+void OpenXRAPI::set_display_refresh_rate(const real_t p_refresh_rate) {
```

Comment:
We don't use `const` for scalar parameters.

I'm also not sure `real_t` is the right type here.

Reply:
Keep it to `float` as that is in the OpenXR API? I'm kinda used to using `real_t` by default. For some reason we used `double` in the plugin

Reply:
Yeah I'm not sure what should be used, started a discussion on #maintainers.

`Variant::FLOAT` (GDScript `float`) is a `double`, so we could use `double` always for scalars in the scripting API to avoid conversion. But currently we tend to use `float` in most places where double precision isn't _needed_. And then there's a ton of `real_t` usage which is unnecessary (`real_t` is configurable for transforms/rendering purposes, it's not promoting all scalars to doubles since most are already doubles). It's a mess.